### PR TITLE
customizable table name on click events

### PR DESCRIPTION
**Pull request by ogidow**
This PR becomes change the table to send the event

example
```js
var td = new Treasure()
td.trackClicks({ tableName: 'custom_table_name' })
```

### CODE REVIEW

**Comment by ogidow** [comment]
thanks your review!

due to circumstances single database used for multiple services. 
so our table naming rule is prefix the table name. and we need table name is customizable

File: README.md [hunk 1]
```diff
@@ -210,6 +210,7 @@ If the database does not exist and you have permissions, it will be created for
 * **config.development** : Boolean (optional) - triggers development mode which causes requests to be logged and not get sent. Default: `false`
 * **config.logging** : Boolean (optional) - enable or disable logging. Default: `true`
 * **config.globalIdCookie** : String (optional) - cookie td_globalid name. Default: `_td_global`
+* **config.globalIdCookie** : String (optional) - name of the table to which the click event will be sent. Default: `clicks`
```
Comment (MikeMcElroy) [inline README.md]: I think you mean to call this `clickTable`, not duplicate `globalIdCookie`.

Comment (ogidow) [reply README.md]: oops! 
i'm so sorry, You are right!

**Comment by ogidow** [comment]
sounds good.
if that is the case, Implementation is like ab02d7b ?

**Comment by MikeMcElroy** [comment]
Sorry for the late reply, @ogidaw!
I think the best implementation would be like that, yes, but with the following addition at [this line](https://github.com/treasure-data/td-js-sdk/blob/ab02d7b3b00f9f05c120398b016995fecc68d8d1/lib/plugins/clicks.js#L18):
```js
  var options = assign({
    element: window.document,
    extendClickData: defaultExtendClickData,
    ignoreAttribute: 'td-ignore',
    tableName: 'clicks'
  }, trackClicksOptions)
```

So the interface would actually be:
`td.trackClicks({ tableName: 'custom_table_name' })`

This way, we wouldn't really need [this line](https://github.com/treasure-data/td-js-sdk/commit/ab02d7b3b00f9f05c120398b016995fecc68d8d1#diff-785e153bf855122a219d0e893f6fd44aR44)

**Comment by ogidow** [comment]
@MikeMcElroy 
I implemented it based on your advice!

**Comment by MikeMcElroy** [comment]
Thanks for your contribution, @ogidow!  I anticipate us doing a release next week of this functionality!

**Pull request merged**